src/main/java/org/folio/rest/client/OkapiClient.java

```diff
@@ -72,17 +74,17 @@ public OkapiClient(Vertx vertx, Map<String, String> okapiHeaders) {
       int responseStatus = response.statusCode();
       if (responseStatus != 200) {
         String errorMessage = String.format("Failed to fetch %s by ID: %s. Response: %d %s",
-          responseType.getName(), id, responseStatus, response.bodyAsString());
+          responseType.getName(), id, responseStatus, response.bodyAsString().replaceAll(LINE_SEPARATOR, EMPTY));
         log.error(errorMessage);
         return failedFuture(new EntityNotFoundException(errorMessage));
       } else {
         try {
           T fetchedObject = objectMapper.readValue(response.bodyAsString(), responseType);
-          log.info("Fetched by ID: {}/{}. Response body: \n{}", path, id, response.bodyAsString());
+          log.info("Fetched by ID: {}/{}. Response body: {}", path, id, response.bodyAsString().replaceAll(LINE_SEPARATOR, EMPTY));
```

Comment:
Please change logging level for bodies of successful responses from `info` to `debug`.

Reply:
It is done, thank you